### ISSUE TITLE
Documentation: Default Control Mappings not tagged with <C-[character]> in Help Files

### ISSUE THREAD

**Issue by MaximoMachado** [vim/vim#7780]
Actual behavior

Typing :help \<C-d> brings up no results

Expected behavior

It should bring up the help page for Ctrl-d

It's a rather minor issue but it'd be convenient if typing what I would type into my .vimrc to do keybinds would also be the same thing I could type for the help command to bring me to its default mapping.

**Comment by chrisbra** [comment]
The convention in Vims help pages for the Ctrl key is to type them as CTRL, so you need to type `:h CTRL-D`

**Comment by chrisbra** [comment]
Perhaps we can amend the documentation to use the `<C-key>` notation. But this will be a huge amount of work to go through the whole documentation and add those tags.

**Comment by MaximoMachado** [comment]
I agree that it would be a huge undertaking. There are some cases where the `<C-key>` notation is used in the help documentation, most notably `<C-Del>` will bring up a result but `CTRL-Del` does not.

**Comment by nickspoons** [comment]
Rather than modifying documentation, couldn't this notation be recognised by the `:help` command, and converted to the "correct" format? This is presumably how shortcuts like `:help ^d` (equivalent to `:help CTRL-D`) work.

**Comment by brammool** [comment]
Typing "<C-x>" is more difficult than "CTRL-x".  Thus I would not want to switch to that or even recommend typing it.
Tags for special keys do use that form, e.g. <C-End>.  That can be a bit confusing.  CTRL-x is used for single letters.
Having the :help command do the conversion should work, and then make sure the CTRL-x form is always available.
